packages/app/components/modalWebView.component.tsx

```diff
@@ -2,10 +2,10 @@ import { useApi } from '@skolplattformen/api-hooks'
 import { Text } from '@ui-kitten/components'
 import URI from 'jsuri'
 import React, { useEffect, useState } from 'react'
-import { Modal, StyleSheet, TouchableOpacity, View } from 'react-native'
+import { Linking,Modal, StyleSheet, TouchableOpacity, View,SyntheticEvent } from 'react-native'
```

Comment:
There are a lot of styling errors in this file. Just use the linting tool (yarn lint --fix) to correct them.